sql/hive/src/test/scala/org/apache/spark/sql/hive/ParquetHiveCompatibilitySuite.scala

```diff
@@ -42,6 +52,15 @@ class ParquetHiveCompatibilitySuite extends ParquetCompatibilityTest with TestHi
        """.stripMargin)
   }
 
+  override def afterEach(): Unit = {
```

Comment:
I was probably just being a little paranoid, perhaps I had missed a `withTable` somewhere.  In the current code, things work just fine if I remove them.